### PR TITLE
Trivial fixes

### PR DESCRIPTION
**Pull request by rhysd**
興味深い記事の翻訳ありがとうございました．いくつか細かいミスを発見したので修正してみましたが，いかがでしょうか？

Closes #10

### CODE REVIEW

File: 3-building-a-cross-platform-mobile-team.md [hunk 1]
```diff
@@ -45,15 +45,15 @@ React Nativeで開発をしていて、避けることのできないバグや
 
 ### チーム分割
 
-NativeとReact Nativeで開発を進めているチームはしばしば技術とコミュニケーション両方の問題にぶつかります。NativeとReact Nativeのコードを分割してしまえば、コードの断片化が発生します。ロジック、モデル、状態の共通化はよりチャレンジングになりエンジニアは全体のflowを一貫して受け持つ事になり、彼らの得意分野の力を出せなくなります。私達はこれらの問題は最初は起こるがWebチームとの共同により徐々にましになっていくだろうと期待していました。が、実際には幾つかのチームがコードや知識のシェアをWeb-Mobile間で始めただけで殆どのチームは潜在的な利益に気づかないままでした。
+NativeとReact Nativeで開発を進めているチームはしばしば技術とコミュニケーション両方の問題にぶつかります。NativeとReact Nativeのコードを分割してしまえば、コードの断片化が発生します。ロジック、モデル、状態の共通化はよりチャレンジングになりエンジニアは全体のフローを一貫して受け持つ事になり、彼らの得意分野の力を出せなくなります。私達はこれらの問題は最初は起こるがWebチームとの協働により徐々にましになっていくだろうと期待していました。が、実際には幾つかのチームがコードや知識のシェアをWeb-Mobile間で始めただけで殆どのチームは潜在的な利益に気づかないままでした。
```
Comment (rhysd) [inline 3-building-a-cross-platform-mobile-team.md:48]: Type Checker の flow と紛らわしいかな？と感じたので，細かいですが日本語のフローに直してみました

**Comment by hotchemi** [comment]
@rhysd Sorry for the late and thank you for your dedicated work 🙇 We've gotten so many PR and eventually the change is conflicted 😢Could you possibly address that?🙇

**Comment by rhysd** [comment]
> Sorry for the late and thank you for your dedicated work

Oh, late? It's just 50mins after creating this PR :wink:

> Could you possibly address that?

Of course. I'll update this branch with resolving conflicts.

**Comment by rhysd** [comment]
@hotchemi I rebased onto the latest master branch. Could you review this PR again?

**Review by hotchemi — APPROVED** [review]
Thank you~!

**Pull request merged**